packages/headless/src/features/question-answering/question-answering-actions-loader.ts

```diff
@@ -50,6 +58,48 @@ export interface QuestionAnsweringActionCreators {
    * @returns A dispatchable action.
    */
   closeFeedbackModal(): PayloadAction;
+  /**
+   * Expand the specified snippet suggestion
+   *
+   * @param payload - The action creator payload.
+   * @returns A dispatchable action.
+   */
+  expandSmartSnippetRelatedQuestion(
+    payload: QuestionAnsweringUniqueIdentifierActionCreatorPayload
+  ): PayloadAction<
+    | QuestionAnsweringUniqueIdentifierActionCreatorPayload
+    | QuestionAnsweringDocumentIdActionCreatorPayload
+  >;
+  /**
+   * Collapse the specified snippet suggestion
+   *
+   * @param payload - The action creator payload.
+   * @returns A dispatchable action.
+   */
+  collapseSmartSnippetRelatedQuestion(
```

Comment:
`expandSmartSnippetRelatedQuestion` and `collapseSmartSnippetRelatedQuestion` weren't exported in the actions loader, so I added them here.

Since they were never exported in the first place, I chose not to expose the true payload type of the action (`QuestionAnsweringUniqueIdentifierActionCreatorPayload | QuestionAnsweringDocumentIdActionCreatorPayload`) by removing the deprecated type.